src/couch/rebar.config.script

```diff
@@ -126,7 +126,7 @@ end.
         };
     {unix, darwin} when SMVsn == "60" ->
         {
-            "-DXP_UNIX -I/usr/include/mozjs-60 -I/usr/local/include/mozjs-60 -std=c++14",
+            "-DXP_UNIX -I/usr/include/mozjs-60 -I/usr/local/include/mozjs-60 -std=c++14 -Wno-invalid-offsetof",
```

Comment:
The `no-invalid-offsetof` is needed because version we use doesn't configure it properly (see [where it was finally autoconfigured](https://github.com/mozilla/gecko-dev/commit/6d93e1b2e447a48303fcec5b10941f2143f625d1#diff-bb21aa33a3f69ccb36c68b220f40ad08f29b9cd2c05dfedae7b9e3d5d4d08f6bL1599)